src/pages/contact.md

```diff
@@ -1,14 +1 @@
----
-title: Contact
----
-Kijk eerst bij de veelgestelde vragen (FAQ) of uw vraag hier tussen staat.
-
-Voor vragen over uw licenties belt u met het RVO loket: 
-
-088 - 042 42 42. 
-
-Dit kan van maandag tot en met vrijdag: van 8.30 tot 17.00 uur. U belt tegen lokaal tarief.

-
-Vanuit het buitenland belt u +31 88 - 042 42 42. U belt dan tegen het tarief dat geldt voor het buitenland.
-
-U kunt contact opnemen met Bureau Erkenningen via <mailto:secretariaat@erkenningen.nl>.
+---
title: Contact
---

Kijk eerst bij de veelgestelde vragen (FAQ) of uw vraag hier tussen staat.

Voor vragen over uw licenties belt u met het RVO loket:

088 - 042 42 42.

Dit kan van maandag tot en met vrijdag: van 8.30 tot 17.00 uur. U belt tegen lokaal tarief.

Vanuit het buitenland belt u +31 88 - 042 42 42. U belt dan tegen het tarief dat geldt voor het buitenland.

U kunt contact opnemen met Bureau Erkenningen via <mailto:secretariaat@erkenningen.nl>.

```

Comment:
This one is weird and not good. Please review. See the preview here (https://deploy-preview-4--erkenningencontentsite.netlify.com/contact) which is awesome btw)...